### PR TITLE
Feature/10216 new repository structure

### PR DESCRIPTION
**Pull request by mayrbenjamin92**
This Pull Request implements changes to essentially get rid of the old folder structure in our current (3.X) Kubernetes repository.

### CODE REVIEW

**Comment by mayrbenjamin92** [comment]
Please add binaries for Windows and Mac as well

File: initialization/src/utils.ts [hunk 1]
```diff
@@ -50,35 +50,35 @@ function createDummySecret() {
 	return 'aaa';
 }
 
-/**
- * Checks whether the '/core/secrets' folder does already exist.
- * If not, the path will be created.
- */
+/*
+* Checks whether the '/core/secrets' folder does already exist.
```
Comment (monty0157) [inline initialization/src/utils.ts]: Comment is outdated

Comment (monty0157) [reply initialization/src/utils.ts]: You seem to be referencing core/secrets in comments throughout this file. Please fix all occurences :)

Comment (mayrbenjamin92) [reply initialization/src/utils.ts]: fixed

**Pull request merged**